src/test/java/api/support/matchers/OverdueFineMatcher.java

```diff
@@ -1,39 +1,56 @@
 package api.support.matchers;
 
 import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasNoJsonPath;
+import static org.folio.circulation.support.JsonPropertyFetcher.getUUIDProperty;
 import static org.hamcrest.core.Is.is;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import java.util.UUID;
 
 import org.folio.circulation.domain.representations.ItemProperties;
+import org.folio.circulation.domain.representations.LoanProperties;
 import org.folio.circulation.support.http.client.IndividualResource;
 import org.hamcrest.Matcher;
 
 import io.vertx.core.json.JsonObject;
 
 public class OverdueFineMatcher {
 
-  public static Matcher<JsonObject> isValidOverdueFine(IndividualResource loan,
+  public static Matcher<JsonObject> isValidOverdueFine(JsonObject loan,
     IndividualResource item, UUID servicePointId, UUID ownerId, UUID feeFineId, Double amount) {
-    return JsonObjectMatcher.allOfPaths(
+
+    List<Matcher<? super String>> matchers = new ArrayList<>(Arrays.asList(
       hasJsonPath("ownerId", is(ownerId.toString())),
       hasJsonPath("feeFineId", is(feeFineId.toString())),
       hasJsonPath("amount", is(amount)),
       hasJsonPath("remaining", is(amount)),
       hasJsonPath("feeFineType", is("Overdue fine")),
       hasJsonPath("feeFineOwner", is("fee-fine-owner")),
-      hasJsonPath("title", is(loan.getJson().getJsonObject("item").getString("title"))),
+      hasJsonPath("title", is(loan.getJsonObject("item").getString("title"))),
       hasJsonPath("barcode", is(item.getJson().getString("barcode"))),
       hasJsonPath("callNumber", is(item.getJson().getJsonObject("effectiveCallNumberComponents")
         .getString("callNumber"))),
       hasJsonPath("location", UUIDMatcher.is(servicePointId)),
       hasJsonPath("materialTypeId", is(item.getJson().getString(ItemProperties.MATERIAL_TYPE_ID))),
-      hasJsonPath("materialType", is(loan.getJson().getJsonObject("item")
+      hasJsonPath("materialType", is(loan.getJsonObject("item")
         .getJsonObject("materialType").getString("name"))),
-      hasJsonPath("loanId", UUIDMatcher.is(loan.getId())),
-      hasJsonPath("userId", is(loan.getJson().getString("userId"))),
-      hasJsonPath("itemId", UUIDMatcher.is(item.getId()))
-    );
+      hasJsonPath("loanId", UUIDMatcher.is(getUUIDProperty(loan, "id"))),
+      hasJsonPath("userId", is(loan.getString("userId"))),
+      hasJsonPath("itemId", UUIDMatcher.is(item.getId())),
```

Comment:
Similarly, I would likely statically import `UUIDMatcher` if possible here

Reply:
If I import `UUIDMatcher.is` statically it'll start to conflict with another `is` from `org.hamcrest.core.Is.is`